packages/components/src/select-control/index.tsx

```diff
@@ -1,8 +1,10 @@
 /**
  * External dependencies
  */
-import { isEmpty, noop } from 'lodash';
+import { isEmpty, noop, omit } from 'lodash';
```

Comment:
Just realised this — can the `omit` import be removed now? I don't see any further usages.
We may want to apply this change directly in #33696